app/src/main/java/com/yacgroup/yacguide/utils/ContactUtils.kt

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2021 Christian Sommer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.yacgroup.yacguide.utils
+
+import android.content.Intent
+import android.net.Uri
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.FileProvider.getUriForFile
+import java.io.File
+import java.io.FileOutputStream
+import java.text.SimpleDateFormat
+import java.util.*
+import kotlin.collections.ArrayList
+
+import com.yacgroup.yacguide.R
+
+class ContactUtils(activity: AppCompatActivity) : ActivityUtils(activity) {
+
+    private var _sdkVersion: Int = android.os.Build.VERSION.SDK_INT
+    private var _androidRelease: String = android.os.Build.VERSION.RELEASE
+
+    fun reportImportError(errMsg: String, jsonFile: Uri) {
```

Comment:
Why do you use two separate methods for reporting import error and bug? Most of the method bodies are equal.

Reply:
I can try to merge but I have the feeling that this results in a more complex code.